.nvmrc

```diff
@@ -0,0 +1 @@
+v18.15.0
```

Comment:
@ggutenberg This change is out of scope. Can we remove it? And open another pull request to upgrade node to `v18.15`

I think there are other places where like
- We should change the node versions like Github workflows.
- We should also check does it impact the package.lock file version.

Reply:
No problem. I didn't see any indication of node version in `package.json` though, so I wasn't sure which node version this was supposed to work with.